api/projects/migrations/0023_rename_identity_overrides_v2_migration_status_project_edge_v2_migration_status.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.25 on 2024-05-02 12:07
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("projects", "0022_add_stale_flags_threshold_to_project"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="project",
+            old_name="identity_overrides_v2_migration_status",
+            new_name="edge_v2_migration_status",
+        ),
+    ]
```

Comment:
Erk, we'll get schema mismatch issues here during deployment! We'll probably have to just rename the python attribute and set the `column_name` on the field definition to keep it the same as it is currently.

Reply:
Done.